src/staticanalysis/bot/requirements.txt

```diff
@@ -1,12 +1,4 @@
 -e ./../../../lib/cli_common[log,pulse,taskcluster,mercurial] #egg=mozilla-cli-common
 
-RBTools
-PyYAML
-parsepatch
-codespell
-yamllint
 datadog
-
-# Fixed versions
-six==1.10.0
-pyasn1==0.3.4
+parsepatch
```

Comment:
Nit: move it back to the previous spot, so it doesn't show in the diff as removed and added